plugins/module_utils/http_url.py

```diff
@@ -0,0 +1,43 @@
+import urllib
+from urllib import error as error
+import urllib.request
+import ssl
+import json
+from string import Template
+import types
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_text
+from ansible.module_utils.urls import open_url
+from ..module_utils.http_request import http_get
+from requests import HTTPError
+import http.client
+from requests.structures import CaseInsensitiveDict
+
+"""{
+      "hostname": ""
+      "port": ""
+      "use_certificate": ""
+      "api_version": ""
+    }
+"""
+HTTPS="https://"
+HTTP= "http://"
+API_ASSISTED_INSTALLER = "api.openshift.com/api/assisted-install/"
+
+def create_url(endpoint) -> str: 
+    if 'api_version' in endpoint.keys():
```

Comment:
This is probably a bad parameter to key off of to decided if the your using the remote as both a probably going to be switching to v2 soon.